src/pages/result/components/filter-button/filter-button.component.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { ROUTES } from '../../../../shared/constants/routes.constants';
 
-import './filter-button.styles.scss';
+import styles from './filter-button.module.scss';
 
 export const FilterButton: React.FC = () => {
   return (
-    <div className="filter-button-container">
-      <Link to={ROUTES.filter} className="filter-button">
-        <span className="filter-button-text">Застосувати фільтр</span>
+    <div className={styles.filterButtonContainer}>
+      <Link to={ROUTES.filter} className={styles.filterButtonContainer__filterButton}>
+        <span className={styles.filterButtonContainer__filterButton__filterButtonText}>Застосувати фільтр</span>
```

Comment:
Element inside element is a bad practice in BEM. It is a common mistake who start to use BEM. Let's keep it for a while. I will explain how to avoid such cases.

Reply:
Okay, I understood!